Terminal.Gui/Core/View.cs

```diff
@@ -170,6 +172,28 @@ internal enum Direction {
 		/// </summary>
 		public Rune HotKeySpecifier { get => textFormatter.HotKeySpecifier; set => textFormatter.HotKeySpecifier = value; }
 
+		/// <summary>
+		/// This is the global setting that can be used as a global shortcut to invoke an action if provided.
+		/// </summary>
+		public Key ShortCut {
```

Comment:
Minor nit, but  "shortcut" is a word. Thus this should be `Shortcut` not `ShortCut`.